web_hosting_manager/app/utils/app_utils.js

```diff
@@ -4,6 +4,29 @@ export const trimErrorMsg = (msg) => {
   return msg.slice(index).trim()
 };
 
+export const parseErrorMsg = (err) => {
+  switch(err.code) {
+    case -102: {
+      return 'Account already exist';
+    }
+    case -107: {
+      return 'Entry already exist';
+    }
+    case -108: {
+      return 'Mutable data maximum size reached';
+    }
+    case -109: {
+      return 'Key not found';
+    }
+    case -113: {
+      return 'Network operation has reached its maximum';
```

Comment:
I think it's not clear what limit has been reached here, perhaps explain that it's about the credit/balance limit for storing data.